opentelemetry/TrackDependency/src/main/java/com/example/TrackDependency.java

```diff
@@ -0,0 +1,54 @@
+package com.example;
+
+import com.azure.monitor.opentelemetry.exporter.AzureMonitorExporterBuilder;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TrackDependency {
+
+  private static final Logger logger = LoggerFactory.getLogger(TrackDependency.class);
+  private static final Tracer spanTracer = initTracer();
```

Comment:
instead of embeding it below, then will make it easier to reference from README.md without referencing specific line number which could get out-of-sync without us noticing/remembering

```suggestion
  private static final Tracer spanTracer = initTracer();

  private static final String CONNECTION_STRING = "<Your Connection String>"
```